seatunnel-api/src/main/java/org/apache/seatunnel/api/table/event/AlterTableColumnEvent.java

```diff
@@ -19,10 +19,15 @@
 
 import org.apache.seatunnel.api.table.catalog.TableIdentifier;
 
+import lombok.Getter;
+import lombok.Setter;
 import lombok.ToString;
 
 @ToString(callSuper = true)
 public abstract class AlterTableColumnEvent extends AlterTableEvent {
+
+    @Getter @Setter protected String sourceColumnType;
```

Comment:
Repeat with this code


https://github.com/apache/seatunnel/blob/476d492165ab2747aa0bb1a5f8a6ed74bfe55f20/seatunnel-api/src/main/java/org/apache/seatunnel/api/table/catalog/Column.java#L82

Reply:
Yeah, I'm working on it.